meta-zephyr-sdk/recipes-devtools/qemu/zephyr-qemu_git.bb

```diff
@@ -194,7 +194,7 @@ inherit autotools pkgconfig
 #--disable-blobs : BIOS needed for x86
 #--disable-fdt: Cannot use if supporting ARM
 
-QEMUS_BUILT = "aarch64-softmmu arm-softmmu i386-softmmu mips-softmmu nios2-softmmu xtensa-softmmu riscv32-softmmu riscv64-softmmu x86_64-softmmu"
+QEMUS_BUILT = "arm-softmmu i386-softmmu mips-softmmu nios2-softmmu xtensa-softmmu riscv32-softmmu riscv64-softmmu x86_64-softmmu"
```

Comment:
lets leave aarch64-softmmu since I expect we'll want it back as cortex-a support progresses.

Reply:
This PR places a symlink to /opt/xilinx-qemu/qemu-system-aarch64 at /usr/bin/qemu-system-aarch64, so it integrates seamlessly to the Zephyr build system.

https://github.com/zephyrproject-rtos/sdk-ng/pull/138/files#diff-69bbdd7ac48e16404691cc2f973c6089R221-R223

At this time, both Cortex-R and Cortex-A (WIP) port are targeting Xilinx Zynq and will be the only user of AArch64 platform (i.e. qemu_cortex_r5 and qemu_cortex_a53 "boards").

Reply:
```
stephanos@DOMINUS:~/Dev/zephyr-sdk-ng$ ls -l /opt/zephyr-sdk/2.2/sysroots/x86_64-pokysdk-linux/usr/bin | grep qemu
-rwxr-xr-x 1 root root    39016 Oct 31 19:33 qemu-edid
-rwxr-xr-x 1 root root  1724312 Oct 31 19:33 qemu-img
-rwxr-xr-x 1 root root  1675352 Oct 31 19:33 qemu-io
-rwxr-xr-x 1 root root  1679896 Oct 31 19:33 qemu-nbd
-rwxr-xr-x 1 root root   616160 Oct 31 19:33 qemu-pr-helper
lrwxrwxrwx 1 root root       45 Oct 31 19:34 qemu-system-aarch64 -> ../../opt/xilinx-qemu/bin/qemu-system-aarch64
-rwxr-xr-x 1 root root 15990000 Oct 31 19:33 qemu-system-arm
-rwxr-xr-x 1 root root 14424608 Oct 31 19:33 qemu-system-i386
-rwxr-xr-x 1 root root 13743376 Oct 31 19:33 qemu-system-mips
-rwxr-xr-x 1 root root  7731040 Oct 31 19:33 qemu-system-nios2
-rwxr-xr-x 1 root root 13238848 Oct 31 19:33 qemu-system-riscv32
-rwxr-xr-x 1 root root 13238848 Oct 31 19:33 qemu-system-riscv64
-rwxr-xr-x 1 root root 14463136 Oct 31 19:33 qemu-system-x86_64
-rwxr-xr-x 1 root root  9705472 Oct 31 19:33 qemu-system-xtensa
```

Reply:
@galak Do you have any request for further changes?

Reply:
Probably want to move the install so its not symlink but just have the binary in the same location as all the others.

Reply:
Can we name the binary something like qemu-system-xilinxarm or something like that?

Reply:
> Probably want to move the install so its not symlink but just have the binary in the same location as all the others.

As far as I am aware, that is not possible because if I also do `--prefix=/usr` for both the upstream and Xilinx QEMU binaries, the shared execs and libraries will conflict (i.e. they will overwrite each other) and can potentially have compatibility issues, especially if upstream and Xilinx QEMU versions are different.

Do you know of a better way to handle this while avoiding conflicts (QEMU doesn't seem to provide a way to prefix or suffix installed binaries)?

Reply:
> Can we name the binary something like qemu-system-xilinxarm or something like that?

So have separate qemu-system-aarch64 for upstream and Xilinx (e.g. qemu-system-aarch64 for upstream and qemu-system-xilinx-aarch64 for Xilinx)?

Reply:
Maybe we tweak the prefix to be /usr/xilinx or something.

Reply:
> So have separate qemu-system-aarch64 for upstream and Xilinx (e.g. qemu-system-aarch64 for upstream and qemu-system-xilinx-aarch64 for Xilinx)?

Yes.

Reply:
> Maybe we tweak the prefix to be /usr/xilinx or something.

Ok, I initially thought of doing that but since that is not the conventional UNIX directory naming practice, I opted to using `/opt` instead.

> > So have separate qemu-system-aarch64 for upstream and Xilinx (e.g. qemu-system-aarch64 for upstream and qemu-system-xilinx-aarch64 for Xilinx)?
> 
> Yes.

Ok. I will still have to symlink `qemu-system-xilinx-aarch64` to `/usr/xilinx/bin/qemu-system-aarch64` though, so as to allow something like this:
```
set(EMU_PLATFORM qemu)
set(QEMU_ARCH xilinx-aarch64)
```

Reply:
Works for me.